Vizzuality/marxan-cloud
# FE (feat): new print layout [MRXN23-296]

## New print layout

### Overview

This PR includes the new version of the general scenario report and removes the previous version.

### Designs

[Figma](https://www.figma.com/file/0xD4gR9pCqU7NROnlSW6ck/Marxan-Layout---Redesign?type=design&node-id=1323-36400&mode=design&t=4yNrUztSGb5RiuQH-0)

### Testing instructions

Select a scenario that has been runned and go to the solution overview section. Download the report. Please make sure that the scenario has been run in a deployed version, not locally.

### Feature relevant tickets

[https://vizzuality.atlassian.net/browse/MRXN23-296](MRXN23-296)
